src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -86,6 +87,7 @@ export class MatProgressSpinner extends _MatProgressSpinnerMixinBase implements
   private _value: number;
   private readonly _baseSize = 100;
```

Comment:
Should this be a module `const`?  Since it is only read during construction, and can't be modified?